CIP-1694/README.md

```diff
@@ -0,0 +1,664 @@
+
+* When in a state of **no-confidence**, the committee can also be replaced via a specific governance action (action 5 below), but this requires the approval of the **SPOs** and the **DReps** instead.
+
+#### Size of the Constitutional Committee
+
+Unlike the Shelley governance design, the size of the Constitutional Committee is not fixed.  It may be changed any time that a new committee is installed.  Likewise, the _quorum_ (the number of votes that are required to enact governance actions) is not fixed and can be varied whenever a new committee is installed. This gives a great deal of flexibility.
+
+### Governance Actions
+
+We define six different types of **governance actions**. A governance action is an on-chain event that is triggered by a transaction and has a deadline after which it cannot be enacted.
+
+An action is said to be **ratified** when it gathers enough votes in its favor (through rules and parameters detailed below). An action that doesn't collect sufficient `yes` votes before its deadline is said to have **expired**.
+An action that has been ratified is said to be **enacted** once it has been activated on the network.
+Regardless of whether they have been ratified, actions may, however, be **dropped** without being **enacted** if, for example, a motion of no confidence is enacted.
+
+
+| Action                                         | Description                                                                                        |
+| ---                                            | ---                                                                                                |
+| 1. Motion of no-confidence                     | A motion to create a _state of no-confidence_ in the current Constitutional Committee  |
+| 2. New Constitutional Committee and/or quorum size | Changes to the members of the Constitutional Committee and/or to its signature threshold   |
+| 3. Updates to the Constitution                 | A modification to the off-chain Constitution, recorded as an on-chain hash of the text document                          |
+| 4. Hard-Fork[^2] Initiation                    | Triggers a non-backwards compatible upgrade of the network; requires a prior software upgrade |
+| 5. Protocol Parameter Changes                  | Any change to one or more updatable protocol parameters, excluding changes to major protocol versions ("hard forks")                                  |
+| 6. Treasury Withdrawals                        | Movements from the treasury, sub-categorized into small, medium or large withdrawals (based on the amount of Lovelace to be withdrawn). The thresholds for treasury withdrawals are discussed below.             |
+
+Any Ada holder can submit a governance action to the chain. They must provide a deposit of `Gov-Deposit` Lovelace, which will be returned when the action is finalized (whether it is **ratified**, has been **dropped**, or has **expired**).
```

Comment:
This petition mechanism can already be implemented via Plutus (assuming it can see proposals), so we don't need to do anything to support that manually. Otherwise, it's not too difficult doing that with an off-chain multisig approach.